SAVE.md

```diff
@@ -0,0 +1,6 @@
+<!-- <div class="search-form">
+  <%= form_tag(search_sites_path, method: :get) do %>
+    <%= text_field_tag 'query', params[:query], placeholder: 'Search for Sites' %>
+    <%= submit_tag 'Search', class: 'alert button expand' %>
+  <% end %>
+</div> -->
```

Comment:
careful

commits with commented out code are now more important than ever to avoid! you will be evaluated on your code organization by employers, and your commits can be looked at

Reply:
Took care of it